scripts/diag.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/sh
+#
+# Copyright (C) 2023 Foundries.io
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Suggested setting with vim:
+# set ai ts=4 sw=4 et
+#
+# Authors:
+#    Tim Anderson <tim.anderson@foundries.io>
+#    Vanessa Maegima <vanessa.maegima@foundries.io>
+#    Caio Pereira <caio.pereira@foundries.io>
+
+VERSION=${VERSION-1.0}
```

Comment:
Does it really need to pass a script version from outside? I think that the version of the script is purely internal property.

Reply:
set it specifically